Swat/SwatViewSelection.php

```diff
@@ -62,35 +53,27 @@ public function __construct(array $selected_items)
         $this->selected_items = array_values($selected_items);
     }
 
-    // }}}
-    // {{{ public function current()
-
     /**
      * Returns the current selected item.
      *
      * @return mixed the current selected item
      */
+    #[ReturnTypeWillChange]
```

Comment:
Could have return type `Traversable<string>` and return `ArrayIterator`

Reply:
if `$selected_items` is `list<string>`, then the `current()` method should just return `string`, no?